hypha/apply/dashboard/views.py

```diff
@@ -216,6 +217,16 @@ def get_context_data(self, **kwargs):
 
         return super().get_context_data(**kwargs)
 
+    def get_my_flagged(self, request, qs):
+        qs = qs.flagged_by(request.user).order_by('-submit_time')
+        row_attrs = dict({'data-flag-type': 'user'}, **SummarySubmissionsTable._meta.row_attrs)
+
+        limit = 5
+        return {
+            'table': SummarySubmissionsTable(qs[:limit], prefix='my-flagged-', attrs={'class': 'all-submissions-table flagged-table'}, row_attrs=row_attrs),
+            'display_more': qs.count() > limit,
+        }
+
```

Comment:
This feels wrong since the exact same function already exist for the admin dashboard.

Reply:
A lot of other methods are duplicated as well like [get_my_submissions](https://github.com/OpenTechFund/hypha/blob/d7181b723de077d311477b58763f7b5d4bdfa8ad/hypha/apply/dashboard/views.py#L196). We will fix it dashboard views refactoring. But okay to duplicate for now.